README.md

```diff
@@ -65,6 +65,7 @@ Now, you can verify it is working by running `gptty --help`. If you experience a
 | max_context_length    | Integer    | 150    |   The maximum length of the input context  |
 | context_keywords_only    | Bool    | True    |   Tokenize keywords to reduce API usage   |
 | preserve_new_lines    | Bool    | False    |   Keep original formatting of response   |
+| verify_internet_endpoint    | String    | "google.com"    |   Address to validate internet connection   |
```

Comment:
Thanks for adding this to the documentation - it will save us time documenting the work you did.